README.md

```diff
@@ -0,0 +1,104 @@
+# Terraform Module: AWS WorkMail DNS Records
+
+> This repository is a [Terraform](https://terraform.io/) Module for managing AWS Route 53 [DNS Records](https://docs.aws.amazon.com/workmail/latest/adminguide/add_domain.html) for AWS WorkMail Organizations.
+
+## Table of Contents
+
+- [Terraform Module: AWS WorkMail DNS Records](#terraform-module-aws-workmail-dns-records)
+  - [Table of Contents](#table-of-contents)
+  - [Requirements](#requirements)
+  - [Dependencies](#dependencies)
+  - [Usage](#usage)
+    - [Module Variables](#module-variables)
+    - [Module Outputs](#module-outputs)
+  - [Author Information](#author-information)
+  - [License](#license)
+
+## Requirements
+
+This module requires Terraform version `0.12.0` or newer.
```

Comment:
`required_version = "> 0.12.0"`

Is this correct?

Reply:
Great catch! Fixed this in https://github.com/operatehappy/terraform-module-aws-route53-workmail-records/pull/1/commits/c9c4b20666f47a34ecd8a0756829e511925f6484